tables_en/1-02.csv

```diff
@@ -42,24 +42,24 @@ nmi,nautical mile,"Conventional unit of length; SI base unit equivalent: 1852 m"
 kt,knot,"Conventional unit of speed, equal to 1 nmi/h; SI base unit equivalent: (1852/3600)m.s-1"
 dB,decibel,"Unit of sound intensity, accepted for use with SI"
 ha,hectare,"Unit of area, accepted for use with SI: 1 ha = 1E4 m2"
-week,week,"Conventional unit of time, equivalent to 604800 s"
-a,year,"Unit of time, equivalent to 31536000 s"
+week,week,"Conventional unit of time, equivalent to 604800 s, as recommeded by SI"

```

Comment:
do we want "as recommended by SI" in the definition?

Reply:
@amilan17 No, this remark by me was misleading. It should be removed, as after "year".